.gitmodules

```diff
@@ -4,9 +4,13 @@
 [submodule "lib/pyexchange"]
 	path = lib/pyexchange
 	url = https://github.com/makerdao/pyexchange.git
-[submodule "lib/ethgasstation-client"]
-	path = lib/ethgasstation-client
-	url = https://github.com/makerdao/ethgasstation-client.git
 [submodule "lib/gdax-client"]
 	path = lib/gdax-client
 	url = https://github.com/makerdao/gdax-client.git
+[submodule "lib/ethgasstation-client"]
```

Comment:
If I understand correctly, the old submodule is still required because these changes are only for the UniswapV2 keeper.

Reply:
Yeah exactly, was trying to minimize the disruption to other DEX keepers.